tests/phpunit/Writer/MSSQLEntrypointTest.php

```diff
@@ -398,10 +405,10 @@ public function testExceptionLogging(): void
         $process = new Process(sprintf('php %s/run.php --data=%s', $this->rootPath, $this->tmpDataPath));
         $process->run();
 
-        $this->assertContains('errFile', $process->getErrorOutput());
-        $this->assertContains('errLine', $process->getErrorOutput());
-        $this->assertContains('trace', $process->getErrorOutput());
-        $this->assertContains('"class":"Keboola\\\\DbWriter\\\\Application"', $process->getErrorOutput());
+        $this->assertContains(
```

Comment:
Tento test ma podla nazvu `testExceptionLogging` testovat logovanie chyb, a ty tu teraz testujes nieco ine .... navrhujem povodny test zachovat a opravit, a vytvorit novy test, napr. `testInputMappingMissing`.

Reply:
jj máš pravdu, je to tam... musel jsem jen původní test upravit a unsetnout storage, aby to vyhodilo správný error